Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.7.4
+MAINTAINER pedge@uber.com
+
+EXPOSE 8080
+RUN \
+  curl -sSL https://get.docker.com/builds/Linux/x86_64/docker-1.12.5 > /bin/docker && \
```

Comment:
This allows the building of the scratch container inside this docker container, check out the makefile rules for it